src/main/fc/fc_msp.c

```diff
@@ -1482,6 +1482,9 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
         motorConfig()->motorPwmProtocol = constrain(sbufReadU8(src), 0, PWM_TYPE_BRUSHED);
 #endif
         motorConfig()->motorPwmRate = sbufReadU16(src);
+        if (dataSize > 7) {
```

Comment:
The use of `dataSize` is a hangover from conversion to using sbuf. So I wouldn't bother fixing it here.

What is required is a separate PR that fixes all uses of `dataSize`.